.kres.yaml

```diff
@@ -2,6 +2,7 @@
 kind: common.Image
 name: image-omni-cloud-provider-qemu
 spec:
+  baseImage: ghcr.io/siderolabs/talosctl:v1.7.5
```

Comment:
This provisioner needs `talosctl`, as the provisioning code imported from Talos requires it to spawn more processes over it. The `talosctl` binary is searched in the working directory and in the PATH by this provisioner. When we use it as the base image, it is found in the current working directory.

Reply:
i think we can just use a copy stage and copy over from the talosctl image

Reply:
I look into that, but it looked like I can copy over from other images (in the same dockerfile), but not from an arbitrary image. Might be wrong though.

Also, does this have any downsides? There is nothing in the talosctl base image other than talosctl.